app/javascript/react/components/ShowContainer.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+class ShowContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      meme: {}
+    };
+  };
+
+  // componentDidMount() {
```

Comment:
Please delete this commented-out code -- pushing commented-out code is bad news bears